stripe-test.html

```diff
@@ -0,0 +1,42 @@
+<html>
+	<head>
+		<style>
+			.alert-danger { color: red; }
+			.hidden { display: none; }
+		</style>
+	</head>
+	<body>
+		<!-- Load Stripe.js on your website. -->
+		<script src="https://js.stripe.com/v3"></script>
+
+		<a class="btn btn-sm btn-danger" href="#" onclick="tip('price_1HGm2hFiF0bSSC08VafHjPIO');return false;">Tip $1</a> 
+		<a class="btn btn-sm btn-info" href="#" onclick="tip('price_1HGm2iFiF0bSSC08ewbTq45M');return false;">Tip $5</a> 
+		<a class="btn btn-sm btn-success" href="#" onclick="tip('price_1HGm2iFiF0bSSC08ZAX8OHae');return false;">Tip $10</a>
+
+		<p id="stripeError" class="alert alert-danger hidden">There was an error sending you to checkout. Please try again.</p>
+
+		<script>
+		  function tip(id){
+		  	var stripe = Stripe('pk_live_TSrb3B6aJLHDffTFBDh5JD3a');
```

Comment:
This is just a test public key, right? Probably OK but either way not sure we want this or any keys in the repo just as a matter of procedure..

Reply:
No it's a real key from their "client-only" checkout solution. Assumed it was okay because it's intended for client only? Not sure how to implement otherwise.